.github/workflows/tests.yml

```diff
@@ -20,16 +20,16 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-          python-version: [3.7, 3.8, 3.9]
+          python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
My sense is that we can cut 3.7 out since it's not supported anymore: https://devguide.python.org/versions/#versions . This movable window will keep the number of tests reasonable.

Reply:
I have seen this pattern as well:
`python-version: ['3.8', '3.11']`

What do you think about that pattern? If tests are passing in 3.11, what are the chances of them failing in 3.9 and 3.10?

Reply:
Alright.